scripts/set_requester_pays.sh

```diff
@@ -1,15 +1,24 @@
 #!/bin/bash
 
 if (( $# < 1 )); then
-  echo "Usage: $0 TERRA_PROJECT_ID [PATH_TO_TERRA_BUCKET_LIST_FILE]"
-  echo "Unless you specify a source file, the script will read buckets from a file 'buckets.txt'."
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "Unless you specify a source file or a string of buckets, the script will read buckets from a file 'buckets.txt'."
   echo "The source file must include newline-delimited Terra bucket paths of format gs://fc-XXXXX"
+  echo 'The string of Terra bucket paths must be formatted as "gs://fc-XXXXX gs://fc-XXXXX gs://fc-XXXXX"'
```

Comment:
the ONLY other thing I can think that would make this even easier on us, is don't require the `gs://` prefix, and if it's not there, put it there in the code
i.e. so i can run this just with `./set_requester_pays.sh project_id fc-12345`